helper.js

```diff
@@ -210,9 +210,9 @@ HELPER.compile = (() => {
 						console.warn( warn.toString() );
 				});
 
-			Fs.writeFileSync( file, prefixed.css );
+				Fs.writeFileSync( file, prefixed.css );
 
-			HELPER.log.success(`Autoprefixed file ${ Chalk.yellow( file ) }`);
+				HELPER.log.success(`Autoprefixed file ${ Chalk.yellow( file ) }`);
```

Comment:
I find it more soothing to know what I'm passing here is going in the right debug hole :)